src/vtep.c

```diff
@@ -190,32 +183,16 @@ static inline void vxlan_build(struct vxlan_hdr *header, uint32_t vni)
 }
 
 /**
- * Compute a hash on the ethernet header that will be used for
- * ECMP/load-balancing
- *
- * @param	eth_hdr the ethernet header that must be hashed
- * @return	the hash of 16 first bytes of the ethernet frame
- */
-static inline  uint16_t ethernet_header_hash(struct ether_hdr *eth_hdr)
-{
-	uint64_t *data = (uint64_t *) eth_hdr;
-	/* TODO: set the seed */
-	uint64_t result = hash_64(data, 8, 0);
-
-	result |= hash_64(data + 1, 8, 0);
-
-	return (uint16_t) result;
-}
-
-/**
- * Compute the udp source port for ECMP/load-balancing
+ * Compute UDP source port to respect good practice from RFC6335:
+ * "the Dynamic Ports, also known as the Private or Ephemeral Ports,
+ *  from 49152-65535 (never assigned)".
  *
- * @param	ether_hash the ethernet hash to use as a basis for the src port
+ * @param	random random number to be converted to correct range.
  * @return	the resulting UDP source port
  */
-static inline uint16_t src_port_compute(uint16_t ether_hash)
+static inline uint16_t src_port_compute(uint16_t random)
 {
-	return (ether_hash % UDP_PORT_RANGE) + UDP_MIN_PORT;
+	return (random % UDP_PORT_RANGE) + UDP_MIN_PORT;
```

Comment:
I don't like random as a variable name, because it's not really random... maybe we should call it seed, as it's use as the seed for the src_port computation ?